src/main/java/com/mabl/integration/jenkins/test/output/Property.java

```diff
@@ -24,6 +24,10 @@ public Property() {
 
     }
 
+    public void setValue(String value) {
```

Comment:
nit: Does property need to be mutable for jenkins screens to work? Seems like we could just create a new object and update it.

Reply:
We don't need the setValue at all. I removed it.